### ISSUE TITLE
`Not` does not work in `@linq` or `@select`

### ISSUE THREAD

**Issue by lassepe** [JuliaData/DataFramesMeta.jl#150]
Currently, it seems like one can not use the `Not` selector in `@linq` or `@select` queries.

For example,

```julia
df = DataFrame(x=rand(3), y=rand(3))
@linq df |> select(Not(:x)) # <-- does not work
@select(df, Not(:x)) # <-- does not work
```
does not work as `:x` is replaced with the values of the `:x` column. 

Instead one needs to escape `:x` in the `Not` call.

```julia
@linq df |> select(Not(^(:x)))
# or...
@select(df, Not(^(:x)))
```

Is this the desired behavior? If so, it may need to be documented (is it?). Are there any cases in which replacing a symbol with the column data is needed in a call to `select`?

**Comment by pdeffebach** [comment]
Thanks for the bug report. This is indeed not desired behavior. 

It will be fixed in upcoming versions of DataFramesMeta. In the meantime I have created #151 to make users informed of this.

**Comment by pdeffebach** [comment]
Marking as 1.X. This is definitely a high priority feature, but it isn't breaking so doesn't need to be added in 1.0.

**Comment by pdeffebach** [comment]
Closed for consolidation with #283
